dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/ConnectorArgumentsTest.java

```diff
@@ -61,11 +62,33 @@ public void getDatabases_trimDatabaseNames() throws IOException {
   @Test
   public void getDatabases_trimDatabaseNamesFilteringOutBlankStrings() throws IOException {
     ConnectorArguments arguments =
-        new ConnectorArguments(
-            new String[] {"--connector", "teradata", "--database", "db1, ,,, db2 "});
+        new ConnectorArguments("--connector", "teradata", "--database", "db1, ,,, db2 ");
 
     List<String> databaseNames = arguments.getDatabases();
 
     assertEquals(ImmutableList.of("db1", "db2"), databaseNames);
   }
+
+  @Test
+  public void testNoUserFlag() throws IOException {
+    ConnectorArguments arguments = new ConnectorArguments("--connector", "abcABC123");
+    Runnable action = () -> arguments.getUserOrFail();
```

Comment:
The invocation of `getUserOrFail` method is part of the `Act` step. I would suggest moving it there, so that it is placed syntactically as the `assertThrows` argument instead of being extracted to the variable of the `Runnable` type.